cuegui/cuegui/HostMonitorTree.py

```diff
@@ -69,48 +69,49 @@ def __init__(self, parent):
                            "indication that a frame is using more cores than it\n"
                            "reserved. If all cores are reserved and the percentage\n"
                            "is way below 100% then the cpu is underutilized.")
-        self.addColumn("Swap", 60, id=4,
+        self.addColumn("Swap", 70, id=4,
                        data=lambda host: cuegui.Utils.memoryToString(host.data.free_swap),
                        sort=lambda host: host.data.free_swap,
                        delegate=cuegui.ItemDelegate.HostSwapBarDelegate,
                        tip="The amount of used swap (red) vs available swap (green)")
-        self.addColumn("Memory", 60, id=5,
+        self.addColumn("Phys Memory", 70, id=5,
```

Comment:
I suggest a change to `Physical Memory` or just `Memory`

Reply:
```suggestion
        self.addColumn("Physical", 70, id=5,
```
Just memory is ambiguous. I'm chaging it to Physical